docs/docs/adapters/prisma.md

```diff
@@ -33,6 +33,10 @@ export default NextAuth({
 })
 ```
 
+:::note
```

Comment:
Instead of the note, let's just change:

https://github.com/nextauthjs/next-auth/blob/ba333f3430483d4df363929f1ca0297492ffa3bb/docs/docs/adapters/prisma.md?plain=1#L23

To:

https://github.com/nextauthjs/next-auth/blob/879faf9fab29436b8b58ba44a0db0391355753b2/apps/dev/pages/api/auth/%5B...nextauth%5D.ts#L49-L50